src/plugin.ts

```diff
@@ -45,6 +48,9 @@ const plugin: FastifyPluginAsync<AppsPluginOptions> = async (fastify, options) =
 
   const aDS = new AppDataService();
   fastify.decorate('appDataService', aDS);
+  const aAS = new AppActionService();
```

Comment:
instead of declaring this here, you can place it inside the the AppAction's service-api.ts, since it's not really necessary in this scope. same thing goes for the FastifyInstance change above.